quic/server/state/ServerStateMachine.h

```diff
@@ -112,6 +113,7 @@ struct QuicServerConnectionState : public QuicConnectionStateBase {
   // Address with higher index is more recently used.
   std::vector<folly::IPAddress> tokenSourceAddresses;
 
+  std::shared_ptr<ServerHandshakeFactory> handshakeFactory;
```

Comment:
Do we need to keep the factory once we get the ServerHandshake out of it?

Reply:
It is necessary on the client side, but so far not on the server side. I am not sure it will be at the end or not. I will remove it for now.